DP3TApp/SharedUI/Style/NSLabelType.swift

```diff
@@ -13,19 +13,19 @@ import UIKit
 class NSFontSize {
     private static let normalBodyFontSize: CGFloat = 16.0
 
-    public static let bodyFontSize: CGFloat = {
+    public static func bodyFontSize() -> CGFloat {
         // default from system is 17.
         let bfs = UIFontDescriptor.preferredFontDescriptor(withTextStyle: UIFont.TextStyle.body).pointSize - 1.0
 
         let preferredSize: CGFloat = normalBodyFontSize
         let maximum: CGFloat = 1.5 * preferredSize
         let minimum: CGFloat = 0.5 * preferredSize
-
+        print(bfs)
```

Comment:
@bastianmorath das print hat es wohl aus versehen in den PR geschafft ;)

Reply:
Ist nun weg ;)